packages/cli-core/src/format/format.ts

```diff
@@ -0,0 +1,19 @@
+import * as os from 'node:os';
+
+/**
+ * Provides formatting utilities.
+ */
+export const format = {
+  list: (lines: string[]) =>
+    lines.map((line: string) => ` - ${line}`).join(os.EOL),
+  indent: (message: string, indentLevel: number) => {
+    if (indentLevel < 0) {
+      return message;
+    }
+    const spaces = ' '.repeat(indentLevel);
+    return message
+      .split('\n')
```

Comment:
to make this platform agnostic, we should replace `os.EOL` with `\n`, then split at `\n` and rejoin with `os.EOL`